ports/nrf/boards/ohs2020_badge/mpconfigboard.mk

```diff
@@ -0,0 +1,10 @@
+USB_VID = 0x239A
+USB_PID = 0x8072
```

Comment:
Please request a new PID by filing an issue, here's an example: #2519

Reply:
https://github.com/adafruit/circuitpython/issues/2529

Reply:
Hopefully resolved with the latest commit.